Gruntfile.js

```diff
@@ -508,9 +509,13 @@ module.exports = function(grunt) {
         cmd:
           'mkdir -p tests/logs && ' +
           './node_modules/.bin/webdriver-manager update && ' +
-          './node_modules/.bin/webdriver-manager start > tests/logs/webdriver.log & ' +
+          './node_modules/.bin/webdriver-manager start & ' +
```

Comment:
Does this mean that running e2e tests locally will mix all output from the webdriver manager with the output of protractor?

Reply:
I reverted this. It should not have been removed.